x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/resolver.tsx

```diff
@@ -33,5 +33,6 @@ export const AlertDetailResolver = styled(
   width: 100%;
   display: flex;
   flex-grow: 1;
-  min-height: 500px;
+  /* gross demo hack */
+  min-height: calc(100vh - 505px);
```

Comment:
❔ Did you check this on laptop-sized monitors? Seems like it would get squishy,

Reply:
that 505px is the static height of the alert detail content above it, not sure what choice we have really.

Reply:
I think this is an improvement, but the long term solution should involve flexbox or something